Description: Disable FetchContent module
 No online operations are permitted during build package.
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: Wed, 22 Jan 2020 21:51:33 +0300

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,16 +4,7 @@ project(tl-expected VERSION 1.0.0 LANGUAGES CXX)
 
 option(EXPECTED_ENABLE_TESTS "Enable tests." ON)
 
-include(FetchContent)
-FetchContent_Declare(
-  tl_cmake
-  GIT_REPOSITORY https://github.com/TartanLlama/tl-cmake.git
-)
-FetchContent_GetProperties(tl_cmake)
-if(NOT tl_cmake_POPULATED)
-  FetchContent_Populate(tl_cmake)
-  set(CMAKE_MODULE_PATH ${tl_cmake_SOURCE_DIR} ${CMAKE_MODULE_PATH})
-endif()
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/debian/tl-cmake ${CMAKE_MODULE_PATH})
 include(add-tl)
 
 tl_add_library(expected SOURCES 
